src/stack/Stack.java

```diff
@@ -4,8 +4,21 @@
 import java.util.regex.Pattern;
 
 public class Stack {
+    public static void main(String args[]) throws StackOverFlowException {
+        Stack data = new Stack();
+//        data.push(5);
+//        data.push(10);
+//        data.push(15);
+        data.push(20);
+        data.push(25);
+        System.out.println("Pop - " + data.pop());
+        System.out.println("Pop - " + data.pop());
+        System.out.println("Pop - " + data.pop());
```

Comment:
String literals should not be duplicated

Define a constant instead of duplicating this literal "Pop - " 3 times